tests/helpers.py

```diff
@@ -331,10 +335,23 @@ def create_pvc(
     if size:
         pvc_data['spec']['resources']['requests']['storage'] = size
     ocs_obj = pvc.PVC(**pvc_data)
+    if measure_time:
+        time_before = time.time()
     created_pvc = ocs_obj.create(do_reload=wait)
+    if measure_time:
+        time_after = time.time()
+        t_time = time_after - time_before
+        # 1 second for PVC creation is a requirement:
+        # https://jira.coreos.com/browse/KNIP-627
```

Comment:
need to understand the correlation between the PV size and creation time

Reply:
I'll take care of it in another PR.
Want to have a coverage tor this one first